spec/fixtures/files/vacancy_sources/my_new_term.json

```diff
@@ -1,42 +0,0 @@
-{
```

Comment:
Noticed I missed this fixture removal when removing MyNewTerm legacy code. So used this commit/PR to remove it too.